Content/Guides/syslog-ng-guide-admin/parser-regexp.htm

```diff
@@ -17,7 +17,10 @@ <h1 name="parser-regexp">Regular expression (regexp) parser</h1>
         <MadCap:keyword term="parsers:['Regular expression']">
         </MadCap:keyword>
         <p>The <MadCap:variable name="General.abbrev" /> application can parse fields from a message with the help of regular expressions. This can be also achieved with the <span class="Code">match()</span> filter, by setting the store-matches flag, but the <span class="Code">regexp-parser()</span> offers more flexibility, like multiple patterns and setting the prefix of the created name-value pairs.</p>
-        <p>For more information about regular expressions in <MadCap:variable name="General.abbrev" />, see <MadCap:xref href="regular-expressions.htm"><span style="color: #04aada;" class="mcFormatColor">Regular expressions</span></MadCap:xref>.</p><pre>&lt;PRI&gt;&lt;VERSION&gt; &lt;YYYY-MM-DD&gt; &lt;HH-MM-SS&gt; &lt;PROGRAM&gt; &lt;PID&gt; &lt;MSGID&gt; - [key1:value1; key2:value2; ... ]</pre>
+        <div class="Note">
+            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">regexp-parser()</span> can create additional name-value pairs only if &quot;named capture groups&quot; are used in the regular expression, for example <span class="Code">(?&lt;test_field&gt;\w+)</span>. For more information, see &quot;named capture groups&quot; in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a>.</p>
```

Comment:
```suggestion
            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">regexp-parser()</span> can create additional name-value pairs only if <span class="Code">&quot;named capture groups&quot;</span> are used in the regular expression, for example <span class="Code">(?&lt;test_field&gt;\w+)</span>. For more information, see <span class="Code">&quot;named capture groups&quot;</span> in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a>.</p>
```

Reply:
I don't think named capture groups need any code highlighting. My understanding is to use code highlighting for syslog-ng configuration elements, e.g. driver names, option names, etc.

I would even drop the quoting around named capture groups if you agree.

Reply:
I agree